frontend/vite.config.ts

```diff
@@ -102,6 +102,12 @@ export default defineConfig(({ mode }) => {
             cors: true, // This disables CORS in dev, key for using ngrok (e.g. for testing Slack integration)
             // Configure origin for proper asset URL generation
             origin: 'http://localhost:8234',
+            proxy: {
```

Comment:
this took ages to figure out as stopping me loading the toolbar  
there's no downside to proxying static assets like this  
and it meant i didn't have to figure out why it had stopped